api/Dockerfile

```diff
@@ -52,7 +52,8 @@ RUN openssl req -newkey rsa:4096 -x509 -nodes -keyout /etc/nginx/certs/app.key -
 
 RUN mkdir -p var/cache \
   && mkdir -p var/logs \
-  && chown -R www-data var
+  && chown -R www-data var \
+  && chmod -R 777 var
```

Comment:
Is this some hangover code from another change? I don't think it should be necessary

Reply:
This came about after trying to set the user token in the var/cache folder. I had to manually change the permissions for it to work but as I'm removing this feature, for now, I'll drop this change also.